python/jupyter/src/widget.ts

```diff
@@ -92,15 +111,14 @@ export class ChemiscopeView extends ChemiscopeBaseView {
         // and then inserting this.el inside the HTML document.
         const element = this.el;
 
+        this._updateWarningTimeout();
```

Comment:
Why do we need to call this everytime? I though the point of traitlets in jupyter was to auto-sync such data.

Reply:
we need to call it also at initialization to fetch the value from the traitlet